extensions/hibernate-orm/runtime-dev/src/main/java/io/quarkus/hibernate/orm/dev/ui/HibernateOrmDevJsonRpcService.java

```diff
@@ -151,53 +148,58 @@ public CompletionStage<JsonRpcMessage<String>> executeHQL(
         }
 
         if (Boolean.TRUE.equals(assistant)) {
-            Assistant a = this.assistant.orElse(null);
-            if (a == null || !a.isAvailable()) {
-                return errorDataSet(
-                        "The assistant is not available, please check the Quarkus assistant extension is correctly configured.");
-            }
+            if (this.assistant.isResolvable()) {
```

Comment:
I'd return early to make it clearer we're failing in this case:

```suggestion
            if (!this.assistant.isResolvable()) {
                return errorDataSet(
                        "The Quarkus Assistant is not available, please install the Chappie extension.");
            }
```